### DISCUSSION TITLE
I get SHA errors when running this

### DISCUSSION THREAD

**Discussion by swinsett** [tj-actions/changed-files#2319] [Q&A] [answered]
Hello all!

First, I do like this action, and I am still pretty new to all this. I am running into an issue on first-time runs, where there is no initial release/tags. I am looking for files changed based on tags, but the initial run doesn't have a tag to compare. 

Basically, I am kicking my workflow off by creating a tag, and then only wanting to action based on changed files. Obviously, the first run should be all files, but I get an error of "No previous tag found". This makes sense, as there wasn't a previous tag, but I still need the initial run.

Is the solution to just create a tag, and move on, or is there another way?

**Comment by jackton1** [comment, marked as answer]
Hi @swinsett, I believe the [message related to no tags found](https://github.com/tj-actions/changed-files/blob/aa11897ec4afaea2c563502c017812d493d21a90/src/utils.ts#L904) is a warning and can be ignored; any subsequent tags you create would return a list of changed files between the previous (first tag) and the new tag made.
